### PR TITLE
Add installation via homebrew cask

### PR DESCRIPTION
**Pull request by weichenw**
I have been using your app for awhile now and just created a cask for it. Just thought I add the installation via homebrew.

### CODE REVIEW

**Comment by timche** [comment]
Awesome, thanks a lot! Any reason why it must be prefixed with `timche`?

**Comment by weichenw** [comment]
> Awesome, thanks a lot! Any reason why it must be prefixed with `timche`?

I submitted gmail-desktop as the cask name initially. But was asked by the homebrew reviewers to prefix timche, to avoid looking erroneously official. 

Happy to change it to other prefixes. That was what they recommended.

Thanks

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ _macOS 10.13+, Linux and Windows 8+ are supported (64-bit only)._
 
 [**Download**](https://github.com/timche/gmail-desktop/releases/latest) the latest `.dmg` file.
 
+or via [cask](http://caskroom.github.io/): `brew cask install timche-gmail-desktop`
```
Comment (timche) [inline README.md:33]:
```suggestion
Or with [Homebrew-Cask](https://caskroom.github.io/): `brew cask install timche-gmail-desktop`
```

File: README.md [hunk 2]
```diff
@@ -146,6 +148,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- markdownlint-enable -->
 <!-- prettier-ignore-end -->
+
```
Comment (timche) [inline README.md:151]: This new line seems random and unrelated(?)

**Comment by weichenw** [comment]
> Ah yes, that makes much sense. Minor changes and we are good to go.
> 
> I've released a new version yesterday. What's the procedure to keep the cask up to date? We might also rename this app some time.

I followed this in the past for other cask
https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask

File: README.md (hunk 1 above)
Comment (timche) [inline README.md:33]:
```suggestion
Or with [Homebrew-Cask](https://caskroom.github.io/): `brew cask install timche-gmail-desktop`
```

**Comment by timche** [comment]
Thanks a lot!

**Comment by timche** [comment]
@all-contributors please add @weichenw  for platform

**Pull request merged**